backend/src/main/java/org/backend/ridinglesson/Ridinglesson.java

```diff
@@ -0,0 +1,11 @@
+package org.backend.ridinglesson;
+
+public record Ridinglesson(
```

Comment:
ich glaube, wenn du zwei Wörter zusammen schreibst, sollte das zweite auch groß geschrieben werden
(du kannst die Klassen/Variablen leicht mit Rechts-Klick/Refactor umändern, dann ändern sie sich im ganzen Projekt)
```suggestion
public record RidingLesson(
```

Reply:
auch in RidingLessonController, RidingLessonService und RidingLessonRepo und im Frontend